dotcom-rendering/src/components/ArticleMeta.apps.tsx

```diff
@@ -28,6 +29,8 @@ type Props = {
 	discussionApiUrl: string;
 	shortUrlId: string;
 	isCommentable: boolean;
+	pageId?: string;
+	headline?: string;
```

Comment:
Under what circumstances is it possible for `pageId` or `headline` to be `undefined`? Doesn't `FEArticleType` say they're always guaranteed to be `string`s?

Reply:
Never mind. I see you're saying they only need to be passed for liveblogs, so make them optional and don't pass them for the other layouts 👍.